bindings/rust/bench/historical-perf/bench-past.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# suppress stdout
+exec >/dev/null
+export CARGO_TERM_QUIET=true
+export RUSTFLAGS=-Awarnings
+
+# immediately bail if any command fails
+set -e
```

Comment:
can you move this to right above bin/bash

Reply:
I think the shebang needs to be the first thing, but I can move the `set -e` above the suppressing stdout.